client/src/routes/automation/blocklyObject.ts

```diff
@@ -0,0 +1,53 @@
+import type { ObjectBlock } from '../../../types/objectsBlockType';
+
+export default class BlocklyObjects {
+    private name: string;
+
+    private type: string;
+
+    private prefix: string;
+
+    private colour: number;
+
+    private options: string[][] = [];
+
+    private toolTip = '';
+
```

Comment:
Detail : why a blank line between each line ?

Reply:
Don't know, it comes from the magical linter

Reply:
Because of the linter rules :)